tests/integration/issues/github_3124/test_cli_executor.py

```diff
@@ -0,0 +1,36 @@
+import os
+import time
+import pytest
+
+import subprocess
+
+cur_dir = os.path.dirname(os.path.abspath(__file__))
+
+
+@pytest.fixture()
+def docker_image():
+    docker_file = os.path.join(cur_dir, 'Dockerfile')
+    os.system(f"docker build -f {docker_file} -t clitest {cur_dir}")
+    time.sleep(3)
+    yield
+    os.system(f"docker rmi -f $(docker images | grep 'clitest')")
+
+
+def test_executor_cli_docker(docker_image):
+    process = subprocess.Popen(
+        ['jina', 'executor', '--uses', 'docker://clitest:latest']
+    )
+    time.sleep(5)
+    poll = process.poll()
+    process.terminate()
+    assert poll is None
+
+
+def test_zed_runtime_cli_docker(docker_image):
+    process = subprocess.Popen(
+        ['jina', 'zed_runtime', '--uses', 'docker://clitest:latest']
+    )
+    time.sleep(5)
+    poll = process.poll()
+    process.terminate()
+    assert poll == 1  # failed
```

Comment:
But why should this fail if you added https://github.com/jina-ai/jina/pull/3144/files#diff-4f725f5fcab143dccca4a394a853b6a44040931ed04346c06e4e3ac041b979c9R215 ?